src/crypto.ts

```diff
@@ -17,19 +21,9 @@ export function HashAll(...args: any[]): Uint8Array {
   return blake.blake2bFinal(h);
 }
 
-// HashRegistryValue hashes the given registry value
-export function HashRegistryValue(registryValue: RegistryValue): Uint8Array {
-  return HashAll(registryValue.tweak, encodeString(registryValue.data), encodeNumber(registryValue.revision));
-}
-
-// HashFileID hashes the given fileID
-export function HashFileID(fileID: FileID): Uint8Array {
-  return HashAll(
-    encodeNumber(fileID.version),
-    encodeString(fileID.applicationID),
-    encodeNumber(fileID.fileType),
-    encodeString(fileID.filename)
-  );
+// Hashes the given registry entry.
+export function HashRegistryEntry(registryEntry: RegistryEntry): Uint8Array {
```

Comment:
this is missing the datakey, you will run into signature verification issues if you do not include it